src/geo/jts.clj

```diff
@@ -1,37 +1,138 @@
 (ns geo.jts
   "Wrapper for the locationtech JTS spatial library. Constructors for points,
   coordinate sequences, rings, polygons, multipolygons, and so on."
+  (:require [clojure.string :as str])
   (:import (org.locationtech.jts.geom Coordinate
+                                      CoordinateSequenceFilter
+                                      Geometry
                                       Point
                                       LinearRing
+                                      LineSegment
+                                      LineString
                                       PrecisionModel
                                       Polygon
-                                      MultiPolygon
                                       PrecisionModel
-                                      GeometryFactory)))
+                                      GeometryFactory)
+           (org.osgeo.proj4j CoordinateTransformFactory
+                             CRSFactory
+                             ProjCoordinate)))
```

Comment:
@willcohen I think there's enough functionality related to projections here to warrant its own namespace; how do you feel about moving all of the new proj functions here into something like `geo.proj`?

Reply:
That's an interesting idea and I'll have to think about that over the weekend. I suspect, however, that it might be impossible to do cleanly because of circular dependencies. Transforming a coordinate requires making a new coordinate, which itself uses the JTS namespace. It may seem silly to make a new namespace if the primary transformation functions are still living in JTS.

Reply:
9dabeda would work! My one concern (and it might not be relevant) is that create-transform will return a proj4j transformation object. proj4j is [currently incubating at locationtech](https://github.com/locationtech/proj4j) (incidentally, this is why I'm currently including the geotrellis version of proj4j. The [original proj4j](https://github.com/Proj4J/proj4j) is getting quite long in the tooth, and the geotrellis version is the most recent version that's available on Maven Central. It's what the locationtech version is based on. That locationtech version [will have a different namespace](https://github.com/locationtech/proj4j/blob/master/src/main/java/org/locationtech/proj4j/CRSFactory.java#L18), like JTS, and I was just trying to hide those objects to avoid an eventual 3.0 version bump due solely to proj4j. It definitely seems strange to me to have the important create-transform be public and live in JTS but the little helpers in proj, which is why I kept it all in the same place.

If, on balance, splitting the namespace and preparing for an eventual version bump seems cleaner than exposing an internal transformation, I'll definitely switch the functions around.